src/popup/views/MnemonicPhrase/index.tsx

```diff
@@ -1,31 +1,31 @@
 import React, { useState } from "react";
 import { shuffle } from "lodash";
-import ConfirmMnemonicPhrase from "popup/components/mnemonicPhrase/ConfirmMnemonicPhrase";
+
+import { EMOJI } from "popup/constants";
+
 import useMnemonicPhrase from "popup/hooks/useMnemonicPhrase";
-import DisplayMnemonicPhrase from "popup/components/mnemonicPhrase/DisplayMnemonicPhrase";
+
 import { Onboarding } from "popup/components/Layout/Fullscreen/Onboarding";
 
+import ConfirmMnemonicPhrase from "popup/components/mnemonicPhrase/ConfirmMnemonicPhrase";
+import DisplayMnemonicPhrase from "popup/components/mnemonicPhrase/DisplayMnemonicPhrase";
+
```

Comment:
Shuffled imports locations based on [this](https://github.com/stellar/product-conventions/blob/master/STYLEGUIDE.md#group-imports-by-location)